flake.nix

```
@@ -348,6 +353,7 @@
             configureFlags = configureFlags ++
               [ "--sysconfdir=/etc" ] ++
               lib.optional stdenv.hostPlatform.isStatic "--enable-embedded-sandbox-shell" ++
+              (if finalAttrs.doCheck then testConfigureFlags else [ "--disable-tests" ]) ++
```

Minor preference, but I do generally like forcing exactly what we want rather than relying on defaulting.
```suggestion
              [ (lib.enableFeature finalAttrs.doCheck "tests") ] ++
              lib.optionals finalAttrs.doCheck testConfigureFlags ++
```